candles/sync_candles.py

```diff
@@ -34,16 +34,16 @@ class BaseSyncCandles(object):
     EXCHANGE = None
     DEFAULT_SYNC_DAYS = 90
     start = end = client = None
-    ALLOWED_DATA_TYPES = ["candles", "futures"]
+    ALLOWED_DATA_TYPES = ["candles", "funding_rates", "futures"]
```

Comment:
I'm confused by this PR in general - I had removed `funding_rates` and switched to a more generic `futures` - in fact it looks like this is merging a lot of things that had changed after my first push(es).